ONNX_VERSION_NUMBER

```diff
@@ -1 +1 @@
-1.1.0
+1.1.2
```

Comment:
Is this really necessary? We should try to remain as permissive as possible in terms of ONNX version number requirements.

Reply:
This causes the CI error.
API changed from 1.1.0 to 1.1.2....
I just want to try if it works.